src/app/components/CodeField/CodeField.tsx

```diff
@@ -8,18 +7,22 @@ import { ScrollSync, ScrollSyncPane } from 'react-scroll-sync';
 export type CodeFieldProps = {
   layout?: 'detail' | 'compact';
   language: string;
+  content: string;
+  onChange: (value: string) => void;
 };
 
 export default function CodeField({
   layout = 'compact',
   language,
+  content,
+  onChange,
 }: CodeFieldProps): JSX.Element {
-  const [content, setContent] = useState('//Your Highlighted Code');
+  // const [content, setContent] = useState('//Your Highlighted Code');
```

Comment:
```suggestion
```

Reply:
this state is uplifted so i terminate the whole line